fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/event/Event.java

```diff
@@ -28,4 +32,16 @@ public int compareTo(Event o) {
 
     return Long.compare(o.getEndTimestamp(), this.getEndTimestamp());
   }
+
+  /**
+   * @return true if event the event takes place today
```

Comment:
misleading? If I understand the code correctly, it is "within a day, hence 24hours" not necessarily "today"